### DISCUSSION TITLE
Welcome to pvm Discussions!

### DISCUSSION THREAD

**Discussion by hjbdev** [hjbdev/pvm#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dhanyg** [comment]
Hi, really appreciate your effort. Will be there a version for linux?

**Reply by hjbdev** [reply]
not sure what benefit pvm would really provide when apt and update-alternatives already exist

open to discussion on it though
